system/Security/CheckPhpIni.php

```diff
@@ -134,12 +135,28 @@ public static function checkIni(): array
             'date.timezone'           => ['recommended' => 'UTC'],
             'mbstring.language'       => ['recommended' => 'neutral'],
             'opcache.enable'          => ['recommended' => '1'],
-            'opcache.enable_cli'      => [],
-            'opcache.jit'             => [],
-            'opcache.jit_buffer_size' => [],
+            'opcache.enable_cli'      => ['recommended' => '1'],
+            'opcache.jit'             => ['recommended' => 'tracing'],
+            'opcache.jit_buffer_size' => ['recommended' => '256', 'remark' => 'Adjust with your free space of memory'],
             'zend.assertions'         => ['recommended' => '-1'],
         ];
 
+        if ($argument === 'opcache') {
+            $items = [
+                'opcache.enable'                  => ['recommended' => '1'],
+                'opcache.enable_cli'              => ['recommended' => '1'],
+                'opcache.jit'                     => ['recommended' => 'tracing'],
```

Comment:
Is it safe to recommend to enable jit?

Reply:
Not safe if using third-party ext like `xdebug`.

Reply:
Based on other and my benchmark, JIT have benefit significant. Moreover, in PHP8.4 a new, better JIT has been implemented

Ref :
- https://github.com/php/php-src/actions/runs/10516802085/job/29139722812
- https://php.watch/articles/jit-in-depth